base/ircd.yaml

```diff
@@ -222,7 +223,11 @@ server:
         # you MUST generate a new one for your installation.
         # suggestion: use the output of `oragono mksecret`
         # note that rotating this key will invalidate all existing ban masks.
-        secret: "CLOAKSECRET_REPLACEME"
+        secret: ""
+
+        # name of an environment variable to pull the secret from, for use with
+        # k8s secret distribution:
+        secret-environment-variable: "ORAGONO_CLOAKING_SECRET"
```

Comment:
Where is this variable created at?

Reply:
It's not yet, @daurminator will hook it up later: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables